_posts/2020-02-14-running-a-great-socratic-seminar.md

```diff
@@ -0,0 +1,70 @@
+
+Once you have established context, discussion can begin. In some cases discussion will commence unprompted, although having some prepared discussion questions can force the attendees to think about the material in way that encourages participation. Not every topic requires discussion, so don't avoid subject matter you think is important because it may not be not be interesting for discussion. You're telling a story, some topics are minor plot devices, others are more central elements of "the conversation."
+
+To that end, there are many topics which can lead to circular, tangential or virulent discussion. As the moderator you should not be afraid of cutting off comments which undermine a respectful atmosphere and/or stymie the forward motion of the event. It is our recommendation that you avoid played out debates as well (e.g. block size, segwit), as they are topics which can be endlessly debated with little benfit or novel insight. Lead by example. Set the tone as the host. If you are disrespectful and don't keep discussion focused you can expect the overall quality of the event to be very low. 
+
+Miscellaneously, we recommend:
+
+- Do not record the event. If you want people to speak freely, do not point the internet's firehose at them. Recording the event will likely lead to self-censorship and limit discussion. 
+- Do introductions. In the beginning of the event give folks 10-20 seconds to say their name and mention a topic that interests them. People may announce other events, job listings, tell jokes and share other things relevant to your members. This brings members closer together, and gives people a chance to get to know each other. 
+- Setup a table at the front of the room so you are at the same level as the crowd, this makes you less of an authority and more of a participant in discussion. 
+- Always bring every possible converter and extension cable you may need (e.g. HDMI extension, VGA to X, DVI to X and HDMI to X)
+- Find a place after the event to gather for drinks and food. Post this on the meetup page so people who cannot make the main event can join for the after party.
+- Manage your time. Respect that many people have families or come from far away and need to get home at a reasonable hour.
+- Short presentations at the end can be very exciting. We recommend following these rules [LINK] to ensure high quality presentations that are relevant to attendees. 
+- Triple check with your event location! Make sure there are no miscommunications on date/time. 
+- If you can find a sponsor to provide food, great! But be careful who you get in bed with--we do not recommend taking sponsorship from ICOs and altcoins for obvious reasons. 
+- Have fun! 
```

Comment:
Maybe another opportunity to reiterate that consistency is what builds an audience. Sticking with a regular meetup while the market is hot and cold reflects the staying power of the meetup.

Reply:
```suggestion
- Consistency is what builds an audience. Sticking with a regular meetup while the market is hot and cold reflects the staying power of the meetup.
- Have fun! 
```